api/db-cdr/changelog-schema/db.changelog-3.1achilles_results_ids.xml

```diff
@@ -0,0 +1,118 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd
+        http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+
+
+  <changeSet author="biopete" id="changelog-3.1 achilles results keys and views" >
+    <preConditions onFail="MARK_RAN">
+      <tableExists tableName="ACHILLES_results" schemaName="cdr" />
+    </preConditions>
+
+
+
+
+    <dropPrimaryKey tableName="ACHILLES_results"/>
+    <dropPrimaryKey tableName="ACHILLES_results_dist"/>
+
+    <addColumn schemaName="cdr" tableName="ACHILLES_results" >
+      <column name="id" type="BIGINT" autoIncrement="true">
+          <constraints primaryKey="true" nullable="false" />
+      </column>
+    </addColumn>
+
+    <addColumn schemaName="cdr" tableName="ACHILLES_results_dist">
+      <column name="id" type="BIGINT" autoIncrement="true">
+        <constraints primaryKey="true" nullable="false" />
+      </column>
+    </addColumn>
+
+    <createIndex
+            indexName="idx_results_analysis_id"
+            tableName="ACHILLES_results"
+            unique="false">
+      <column name="analysis_id"/>
+      <column name="stratum_1"/>
+    </createIndex>
+
+
+    <createIndex
+            indexName="idx_dist_analysis_id"
+            tableName="ACHILLES_results_dist"
+            unique="false">
+      <column name="analysis_id"/>
+      <column name="stratum_1"/>
+
+    </createIndex>
+    <createIndex
+            indexName="idx_strat2_results_analysis_id"
+            tableName="ACHILLES_results"
+            unique="false">
+      <column name="stratum_2"/>
+    </createIndex>
+    <createIndex
+            indexName="idx_strat2_dist_results_analysis_id"
+            tableName="ACHILLES_results_dist"
+            unique="false">
+      <column name="stratum_2"/>
+    </createIndex>
+
+    <createView replaceIfExists="true" schemaName="cdr" viewName="ACHILLES_results_view">
```

Comment:
Where is this view referenced?

Reply:
It was too slow to use.  I ended up just making a table out of it. I could add some delete views.

Reply:
Don't add delete views -- just remove them (and anything else you're not using anymore) from this PR please.

Reply:
optimizing after we finalize db.